app/views/shared/_footer.html.erb

```diff
@@ -0,0 +1,10 @@
+<footer class="footer wrapper">
+  <p>Made for <a href="https://hacktoberfest.digitalocean.com/">Hacktoberfest 2017</a>.</p>
```

Comment:
I was thinking that this might confuse the final users, most of them won't be programmers and this footer would appear on every website created with "cansei" 🤔

Reply:
Good point. Should i completely remove it, or do you prefer we put a setting in the configuration to add a footer message?

Reply:
I think we remove it to try to keep it simple at first. #my2cents

Reply:
Removed!